src/jdk.incubator.jextract/share/classes/jdk/incubator/jextract/tool/resources/Cstring.java.template

```diff
@@ -0,0 +1,83 @@
+// Generated by jextract
+
+import java.lang.invoke.VarHandle;
+import java.nio.charset.Charset;
+import jdk.incubator.foreign.AllocationScope;
+import jdk.incubator.foreign.Foreign;
+import jdk.incubator.foreign.MemoryAddress;
+import jdk.incubator.foreign.MemoryLayout;
+import jdk.incubator.foreign.MemorySegment;
+import static jdk.incubator.foreign.MemoryLayouts.C_CHAR;
+
+public final class Cstring {
+    // don't create!
+    private Cstring() {
+    }
+
+    private static final Foreign foreign = Foreign.getInstance();
+
+    private static VarHandle arrayHandle(MemoryLayout elemLayout, Class<?> elemCarrier) {
+        return MemoryLayout.ofSequence(elemLayout)
+                .varHandle(elemCarrier, MemoryLayout.PathElement.sequenceElement());
+    }
```

Comment:
Could this be moved to a common file (seems to be the same every time)? Maybe you could put it in RuntimeHelper, and have an `import static ${RUNTIME_HELPER}` where you plugin in the generated class name?